### PR TITLE
Join array of shas and pass that as the query param

### PR DESCRIPTION
**Pull request by baristaGeek**
## Description

Previously, we were only passing 1 SHA as the query parameter to the GitHub API when searching for commits based on SHAs. This was limiting the number of results we were showing. With this fix, we now have multiple search results. 

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: src/extension.ts [hunk 1]
```diff
@@ -78,10 +79,18 @@ function getPRsPerSHAs() {
   watermelonPanel.currentPanel?.doRefactor({
     command: "loading",
   });
+
+  // Splice the array to avoid the 256 character restriction on the query to the GitHub API
+  if (arrayOfSHAs.length > 22) {
+    arrayOfSHAs.splice(22, arrayOfSHAs.length - 22);
```
Comment (EstebanDalelR) [inline src/extension.ts:85]: This modifies the array, could we use a copying method such as [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice)

Comment (EstebanDalelR) [reply src/extension.ts:85]: Slice also allows slicing above the number of elements on the array without breaking, making the if innecesary

File: src/extension.ts [hunk 2]
```diff
@@ -78,10 +79,18 @@ function getPRsPerSHAs() {
...
+  }
+
+  let joinedArrayOfSHAs = arrayOfSHAs.join();
```
Comment (EstebanDalelR) [inline src/extension.ts:88]: Had to check, `join` joins with a ","

const elements = ['Fire', 'Air', 'Water'];

console.log(elements.join());
// output: "Fire,Air,Water"``

**Comment by baristaGeek** [comment]
Closing because #54 has the merge conflicts correctly solved

**Pull request closed without merging**